_posts/2016-02-18-starter-issues.md

```diff
@@ -0,0 +1,112 @@
+
+If the issue involves writing code, then we link to a description on how to setup the repository and how to run tests
+
+```
+- [ ] [Setup repository locally](url-here) and make sure that all tests pass
+```
+
+If the issue involves an update to the `README.md` file, then this is the next item, something like this
+
+```
+- [ ] update the [README](link-to-readme-section): add `myLib.myMethod` in the same format the other APIs are documented
+```
+
+At Hoodie, we follow [Commit Message Conventions](https://github.com/hoodiehq/hoodie/blob/master/CONTRIBUTING.md#commit-message-conventions). Instead of explaining why and how, we simply make actionable items for the separate commits we’d like to have
+
+```
+- [ ] Commit with `docs(README): myLib.myMethod`
+```
+
+Once the first commit is ready, it’s time to start the Pull Request. Even though we link the tutorial on how to contribute to open source above, we do it again, because it doesn’t hurt. We also congratulate if this is going to be the first Pull Request for someone, because it means a lot to us :)
+
+```
+- [ ] Start a Pull Request. Set description to `closes #123`
+If this is your first, welcome :tada: :smile: [Here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request)
+```
+
+Once the PR is created, we check all above boxes and replace the above item with something like `- [x] Start a Pull Request (#123)`, where `#123` is the number of the Pull Request.
+
+If this issue is about code, a test needs to be written or adapted. We give guidance where the test needs to be added, ideally like to the files / lines
+
+```
+- [ ] Create a file `/test/unti/my-method-test.js` in [/test/unit](https://github.com/myuser/mylib/tree/master/test/unit). See [/test/unit/my-other-method-test.js](https://github.com/myuser/mylib/tree/master/test/unit/other-method-test.js) for comparison
+- [ ] Add a check that `myLib.myMethod` is a function in [/test/integration/constructor-test.js](https://github.com/myuser/mylib/tree/master/test/integration/constructor-test.js)
+```
+
+Then we ask the user to commit and push the tests. This way the commit will trigger our continuous integration tests on [Travis CI](https://travis-ci.org) and will add a red **×** to the commit, so we can be sure that tests are failing as expected. Here’s [an example](https://github.com/hoodiehq/pouchdb-hoodie-api/pull/100/commits) of what I mean.
+
+```
+- [ ] Make sure that tests fail when run with `npm test`. Commit with `test: myLib.myMethod` and push
+```
+
+Now comes the fun, the “actual” work, and all it takes is an item describing it. If helpful, we link to the right place and explain how the feature / bug fix can be implemented. But it’s okay to keep it short and leave something for the contributor to figure out. It’s fun after all, and at this point the contributor should feel comfortable to ask for help if they get stuck. So often time we simply do:
```

Comment:
absolutely true. Thanks so much for catching this! Also all your other comments are great, incorporating all of them right now :)